listenbrainz/webserver/templates/emails/id_paused.txt

```diff
@@ -0,0 +1,3 @@
+We have paused your ListenBrainz account {{ username }} because of a problem with your account.
+Our community manager will reach out to you with the details as to why we took this step.
+Feel free to Contact Us {{ url }} if you have any questions about this.
```

Comment:
`Contact Us {{ url }}` is weird. Ideally "contact us" (lowercase) would be the link, but that might not be supported by the email system, in which case maybe:

```suggestion
Feel free to contact us (see {{ url }}) if you have any questions about this.
```

Also, shouldn't this have a signature like `id_unpaused`? The same seems fine.